### PR TITLE
Aubriebranch

### PR DESCRIPTION
**Pull request by flowergirlthatcodes3**
Added routes for details button.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -29,116 +30,122 @@ const App = ()=> {
...
     <div>
       {
         auth.id ? (
           <>
-            <div className='navi'>
             <nav>
-            <Link to='/home'>Home</Link>
-
-              <Link to='/products'>Courses ({ products.length })</Link>
+              <Link to='/products'>Products ({ products.length })</Link>
```
Comment (leblatz) [inline src/index.js]: this overrides home link and classname organization:

 <div className='navi'>
            <nav>
            <Link to='/home'>Home</Link>

              <Link to='/products'>Courses ({ products.length })</Link>

File: src/index.js [hunk 2]
```diff
@@ -147,10 +154,8 @@ const App = ()=> {
                 <button onClick={ logout }>Logout</button>
               </span>
             </nav>
-            </div>
             <main>
               <Routes>
-              <Route path='/home' element= {<Home/>}></Route>
```
Comment (leblatz) [inline src/index.js:153]: home route is deleted

File: src/index.js [hunk 3]
```diff
@@ -147,10 +154,8 @@ const App = ()=> {
                 <button onClick={ logout }>Logout</button>
               </span>
             </nav>
-            </div>
```
Comment (leblatz) [inline src/index.js:150]: classname div is deleted

File: src/index.js [hunk 4]
```diff
@@ -184,16 +189,11 @@ const App = ()=> {
             </>
         ):(
           <div>
-              <div className='navi'>
```
Comment (leblatz) [inline src/index.js:187]: classname and home are deleted, similar to previous comment 

  <div className='navi'>
            <nav>   
              <Link to='/home'>Home</Link>
              <Link to='/products'>Courses</Link>

File: src/index.js [hunk 5]
```diff
@@ -184,16 +189,11 @@ const App = ()=> {
...
-            <nav>   
-              <Link to='/home'>Home</Link>
-              <Link to='/products'>Courses</Link>
+            <nav>
               <Link to='/account'>Account</Link>
-              
+              <Link to='/products'>Products</Link>
             </nav>
-            </div>
```
Comment (leblatz) [inline src/index.js:194]: classname div end is deleted

File: src/index.js [hunk 6]
```diff
@@ -184,16 +189,11 @@ const App = ()=> {
...
             <Routes>
-              <Route path='/home' element={<Home />}></Route>
```
Comment (leblatz) [inline src/index.js:196]: home route is deleted

**Review by leblatz — DISMISSED** [review]
commented on all the things your file overrides

**Review by WyldeIce — APPROVED** [review]

**Review by leblatz — APPROVED** [review]

**Pull request merged**